fix: Wrong customer context on login if entry from `sales_channel_api_context` table is expired (#11099)
shopware/platform: changelog/_unreleased/2025-07-09-fix-wrong-customer-context-on-login.md

---
title: Fix wrong customer context on login if entry from sales_channel_api_context is expired
issue: 11097
author: Christoph Pötz
author_github: @acris-cp
---
# Core

> * `Shopware\Core\System\SalesChannel\Context\SalesChannelContextPersister` don't remove customerId and other information from payload if sales_channel_api_context is expired

Reviewer: ```suggestion
* Changed `Shopware\Core\System\SalesChannel\Context\SalesChannelContextPersister` so it doesn't remove `customerId` and other information from payload if the context is expired
```
